Makefile

```diff
@@ -522,11 +522,11 @@ localnet_up: ## Starts up a k8s LocalNet with all necessary dependencies (tl;dr
 
 .PHONY: localnet_client_debug
 localnet_client_debug: ## Opens a `client debug` cli to interact with blockchain (e.g. change pacemaker mode, reset to genesis, etc). Though the node binary updates automatiacally on every code change (i.e. hot reloads), if client is already open you need to re-run this command to execute freshly compiled binary.
-	kubectl exec -it deploy/dev-cli-client --container pocket -- p1 debug --remote_cli_url http://pocket-validators:50832
+	kubectl exec -it deploy/dev-cli-client --container pocket -- p1 debugui --remote_cli_url http://pocket-validators:50832
 
 .PHONY: localnet_shell
 localnet_shell: ## Opens a shell in the pod that has the `client` cli available. The binary updates automatically whenever the code changes (i.e. hot reloads).
-	kubectl exec -it deploy/dev-cli-client --container pocket -- /bin/bash
+	kubectl exec -it deploy/dev-cli-client --container pocket -- /bin/bash -c "export POCKET_REMOTE_CLI_URL=http://pocket-validators:50832; bash"
```

Comment:
Similar question / observation to the above. Is there a reason not to put this in the [cli-client mainfest](https://github.com/pokt-network/pocket/blob/main/build/localnet/manifests/cli-client.yaml#L77) instead?

Again, it looks like it's already set there (to a different value). Additionally, if we're removing `POCKET_P2P_IS_CLIENT_ONLY` we should remove it in [the manifest](https://github.com/pokt-network/pocket/blob/main/build/localnet/manifests/cli-client.yaml#L41) as well.